enhancements/storage/manila-csi-driver-operator.md

```diff
@@ -94,37 +94,37 @@ Actions:
 
 - Go through the lifecycle of volume (create, update, mount, write data, delete, etc.)
 
-#### Write Manila CSI driver operator
+#### Write Manila CSI driver operator (Done)
 
 The operator should be able to create, configure and manage Manila CSI driver for Kubernetes and OpenShift. In other words, automate what has been done on the previous step.
 Actions:
 
-- Create a new repo in OpenShift’s github: github.com/openshift/manila-csi-driver-operator
+- Create a new repo in OpenShift’s github: github.com/openshift/csi-driver-manila-operator
 
-- Implement the driver, using [OpenShift Operator SDK](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html)
+- Implement the operator, using [OpenShift Operator SDK](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html)
 
 ##### CSI driver operator installation
 
-The installation will include next steps:
+Starting from OpenShift 4.5 the installation will include next steps:
 
 - `cluster-version-operator` starts `openshift-cluster-storage-operator`.
 
-- `openshift-cluster-storage-operator` checks it runs on OpenStack and Manila service is available
+- `openshift-cluster-storage-operator` checks if it runs on OpenStack, installs Manila CSI driver operator via OLM, starts it and monitors its status (i.e. monitors existence of Manila CSI driver operator ClusterOperator CR and reports its own `openshift-cluster-storage-operator` status based on it).
 
-- `openshift-cluster-storage-operator` starts Manila CSI driver operator and monitors its status (i.e. monitors existence of Manila CSI driver operator ClusterOperator CR and reports its own `openshift-cluster-storage-operator` status based on it).
-
-- Manila CSI driver operator starts, populates [configuration](https://github.com/kubernetes/cloud-provider-openstack/tree/master/manifests/manila-csi-plugin) and secrets for Manila CSI driver and runs the CSI driver (i.e. starts Deployment with the controller parts and DaemonSet with the node parts). Additionally it creates at least one non-default StorageClass that users can use in their RWX PVCs
+- Manila CSI driver operator checks if Manila service is available. If it's true then it starts, populates [configuration](https://github.com/kubernetes/cloud-provider-openstack/tree/master/manifests/manila-csi-plugin) and secrets for Manila CSI driver and runs the CSI driver (i.e. starts Deployment with the controller parts and DaemonSet with the node parts). Additionally it creates at least one non-default StorageClass that users can use in their RWX PVCs. If Manila service is not available, the operator does nothing.
 
```

Comment:
Looking at https://github.com/openshift/csi-driver-manila-operator, it looks like it uses a single secret object for operator configuration and it is not apparent to me if location (name/namespace) of secret is hardcoded in operator itself or configurable. 

What I would prefer is:
1. Have projectID, domainID (non-secret stuff) as top level fields in manilacsi spec.
2. Allow user to configure location from which username and password can be fetched and hence manilacsi spec should include a secretref field (if it is going to be configurable by the user)
3. manilacsi type should be cluster scoped as proposed in https://github.com/openshift/enhancements/blob/47be93851058b7e39be4563e128cc7c6ad8aaebf/enhancements/storage/csi-driver-install.md
4. while the operator can be installed in any arbitrary namespace, the driver should be installed in a pre-defined/hardcoded namespace.

Reply:
@Fedosin, I think the API of the operator is actually pretty bad - cryptic secret and empty structs in CR itself: https://github.com/Fedosin/csi-driver-manila-operator/blob/5a029987a2094175d400e0e8e93089f716034d09/pkg/apis/manilacsi/v1alpha1/manilacsi_types.go#L8-L13 

However, it was not part of the proposal before and is not even now.

So for *this* PR, I think the change presented here are lgtm.